lib/pf/util.pm

```diff
@@ -751,16 +751,18 @@ sub pf_run {
     my $caller = ( caller(1) )[3] || basename($0);
     $caller =~ s/^(pf::\w+|main):://;
 
+    my $loggable_command = $command;
+    $loggable_command =~ s/$options{log_strip}/*obfuscated-information*/g; 
```

Comment:
It's better to have it as a regex I think since it allows to do finer replacement in the command.

I love dinde